### PR TITLE
6593 payara docs

### LINKED ISSUE #6593: Payara Documentation Updates
As we move towards upgrading app server requirements to Payara 5, we'll need to update our installation and developer guides. We should also do a check of other guides to see if there are any references to glassfish 4.1 there.

### PR DESCRIPTION
**Pull request by pdurbin**
**What this PR does / why we need it**:

Many, many places in the guide reference Glassfish 4. For Dataverse 5 we are moving to Payara 5.

**Which issue(s) this PR closes**:

Closes #6593

**Special notes for your reviewer**:

- ~~Details about a patch to Mojarra (JSF) have not yet been included.~~ Added in b3619996c
- "app server" was used where it makes sense to
- An attempt was made to use a Unix user called "dataverse" instead of "glassfish".
- In the Installation Guide, should we delete the old Two Ravens content? I put a note saying "Please note that in the text below, Glassfish was changed to Payara but not tested."
- ~~When should we update Vagrant code?~~ Started in a7c986d but deferred until #6849
- ~~When should we update docker-aio code?~~ @pameyer said he'd work on this
- ~~When should we update docker-dcm code?~~  @pameyer said he'd work on this
- ~~Should we clear out the "Containers" page in the dev guide and delete the related scripts, Openshift config and Docker files? The content is old and references Glassfish.~~ Done in e72ba3c.

**Suggestions on how to test this**:

- Try to install Dataverse on Payara 5 with these instructions.
- Set up as many extra features (Make Data Count, etc.) as is reasonable.

**Does this PR introduce a user interface change?**:

No

**Is there a release notes update needed for this change?**:

No.

**Additional documentation**:

None.

### CODE REVIEW

File: doc/sphinx-guides/source/admin/harvestclients.rst [hunk 1]
```diff
@@ -24,6 +24,6 @@ The process of creating a new, or editing an existing client, is largely self-ex
 What if a Run Fails?
 ~~~~~~~~~~~~~~~~~~~~
 
-Each harvesting client run logs a separate file per run to Glassfish's default logging directory (``/usr/local/glassfish4/glassfish/domains/domain1/logs/`` unless you've changed it). Look for filenames in the format  ``harvest_TARGET_YYYY_MM_DD_timestamp.log`` to get a better idea of what's going wrong.
+Each harvesting client run logs a separate file per run to Payara's default logging directory (``/usr/local/payara5/glassfish/domains/domain1/logs/`` unless you've changed it). Look for filenames in the format  ``harvest_TARGET_YYYY_MM_DD_timestamp.log`` to get a better idea of what's going wrong.
```
Comment (scolapasta) [inline doc/sphinx-guides/source/admin/harvestclients.rst:27]: Here's one we can change to App server (though the directory still has it)

File: doc/sphinx-guides/source/admin/metadataexport.rst [hunk 2]
```diff
@@ -28,7 +28,7 @@ Note, that creating, modifying, or re-exporting an OAI set will also attempt to
 Export Failures
 ---------------
 
-An export batch job, whether started via the API, or by the application timer, will leave a detailed log in your configured logs directory. This is the same location where your main Glassfish server.log is found. The name of the log file is ``export_[timestamp].log`` - for example, *export_2016-08-23T03-35-23.log*. The log will contain the numbers of datasets processed successfully and those for which metadata export failed, with some information on the failures detected. Please attach this log file if you need to contact Dataverse support about metadata export problems.
+An export batch job, whether started via the API, or by the application timer, will leave a detailed log in your configured logs directory. This is the same location where your main Payara server.log is found. The name of the log file is ``export_[timestamp].log`` - for example, *export_2016-08-23T03-35-23.log*. The log will contain the numbers of datasets processed successfully and those for which metadata export failed, with some information on the failures detected. Please attach this log file if you need to contact Dataverse support about metadata export problems.
```
Comment (scolapasta) [inline doc/sphinx-guides/source/admin/metadataexport.rst:31]: Can probably change this one (i.e. no matter what server we ever use, all logs should be in same place)

File: doc/sphinx-guides/source/admin/monitoring.rst [hunk 3]
```diff
@@ -90,12 +90,12 @@ Process the logs:
...
 
-Please note that the Dataverse team has attempted to parse Glassfish logs using AWStats but it didn't seem to just work and posts have been made at https://stackoverflow.com/questions/49134154/what-logformat-definition-does-awstats-require-to-parse-glassfish-http-access-logs and https://sourceforge.net/p/awstats/discussion/43428/thread/9b1befda/ that can be followed up on some day.
+Please note that the Dataverse team has attempted to parse Glassfish/Payara logs using AWStats but it didn't seem to just work and posts have been made at https://stackoverflow.com/questions/49134154/what-logformat-definition-does-awstats-require-to-parse-glassfish-http-access-logs and https://sourceforge.net/p/awstats/discussion/43428/thread/9b1befda/ that can be followed up on some day.
 
-Database Connection Pool used by Glassfish
-------------------------------------------
+Database Connection Pool used by Payara
```
Comment (scolapasta) [inline doc/sphinx-guides/source/admin/monitoring.rst:95]: app server

File: doc/sphinx-guides/source/admin/monitoring.rst [hunk 4]
```diff
@@ -90,12 +90,12 @@ Process the logs:
...
+---------------------------------------
 
-https://github.com/IQSS/dataverse/issues/2595 contains some information on enabling monitoring of Glassfish, which is disabled by default. It's a TODO to document what to do here if there is sufficient interest.
+https://github.com/IQSS/dataverse/issues/2595 contains some information on enabling monitoring of Payara, which is disabled by default. It's a TODO to document what to do here if there is sufficient interest.
```
Comment (scolapasta) [inline doc/sphinx-guides/source/admin/monitoring.rst:98]: the app server

File: doc/sphinx-guides/source/admin/troubleshooting.rst [hunk 5]
```diff
@@ -97,14 +97,14 @@ followed by an Exception stack trace with these lines in it:
...
 
 
 Constraint Violations Issues
 ----------------------------
 
-In real life production use, it may be possible to end up in a situation where some values associated with the datasets in your database are no longer valid under the constraints enforced by the latest version of Dataverse. This is not very likely to happen, but if it does, the symptomps will be as follows: Some datasets can no longer be edited, long exception stack traces logged in the Glassfish server log, caused by::
+In real life production use, it may be possible to end up in a situation where some values associated with the datasets in your database are no longer valid under the constraints enforced by the latest version of Dataverse. This is not very likely to happen, but if it does, the symptomps will be as follows: Some datasets can no longer be edited, long exception stack traces logged in the Payara server log, caused by::
```
Comment (scolapasta) [inline doc/sphinx-guides/source/admin/troubleshooting.rst:107]: app server

File: doc/sphinx-guides/source/admin/troubleshooting.rst [hunk 6]
```diff
@@ -32,49 +32,49 @@ A User Needs Their Account to Be Converted From Institutional (Shibboleth), ORCI
 
 See :ref:`converting-shibboleth-users-to-local` and :ref:`converting-oauth-users-to-local`.
 
-.. _troubleshooting-glassfish:
+.. _troubleshooting-payara:
```
Comment (scolapasta) [inline doc/sphinx-guides/source/admin/troubleshooting.rst:35]: I'm ok leaving this section as troubleshooting Payara, so all the references inside are fine). (some of this troubleshooting may actually be more generic, but it's all under this header)

File: doc/sphinx-guides/source/developers/coding-style.rst [hunk 7]
```diff
@@ -85,7 +85,7 @@ Use this ``logger`` field with varying levels such as ``fine`` or ``info`` like
 
     logger.fine("will get thumbnail from dataset logo");
 
-Generally speaking you should use ``fine`` for everything that you don't want to show up in Glassfish's ``server.log`` file by default. If you use a higher level such as ``info`` for common operations, you will probably hear complaints that your code is too "chatty" in the logs. These logging levels can be controlled at runtime both on your development machine and in production as explained in the :doc:`debugging` section.
+Generally speaking you should use ``fine`` for everything that you don't want to show up in Payara's ``server.log`` file by default. If you use a higher level such as ``info`` for common operations, you will probably hear complaints that your code is too "chatty" in the logs. These logging levels can be controlled at runtime both on your development machine and in production as explained in the :doc:`debugging` section.
```
Comment (scolapasta) [inline doc/sphinx-guides/source/developers/coding-style.rst:88]: app server

File: doc/sphinx-guides/source/developers/debugging.rst [hunk 8]
```diff
@@ -8,7 +8,7 @@ Debugging
 Logging
 -------
 
-By default, Glassfish logs at the "INFO" level but logging can be increased to "FINE" on the fly with (for example) ``./asadmin set-log-levels edu.harvard.iq.dataverse.api.Datasets=FINE``. Running ``./asadmin list-log-levels`` will show the current logging levels.
+By default, Payara logs at the "INFO" level but logging can be increased to "FINE" on the fly with (for example) ``./asadmin set-log-levels edu.harvard.iq.dataverse.api.Datasets=FINE``. Running ``./asadmin list-log-levels`` will show the current logging levels.
```
Comment (scolapasta) [inline doc/sphinx-guides/source/developers/debugging.rst:11]: app server

File: doc/sphinx-guides/source/developers/testing.rst [hunk 9]
```diff
@@ -168,7 +168,7 @@ Running the full API test suite using Docker
 
 To run the full suite of integration tests on your laptop, we recommend using the "all in one" Docker configuration described in ``conf/docker-aio/readme.txt`` in the root of the repo.
 
-Alternatively, you can run tests against Glassfish running on your laptop by following the "getting set up" steps below.
+Alternatively, you can run tests against Payara running on your laptop by following the "getting set up" steps below.
```
Comment (scolapasta) [inline doc/sphinx-guides/source/developers/testing.rst:171]: the app server

File: doc/sphinx-guides/source/developers/tips.rst [hunk 10]
```diff
@@ -2,48 +2,46 @@
 Tips
 ====
 
-If you just followed the steps in :doc:`dev-environment` for the first time, you will need to get set up to deploy code to Glassfish. Below you'll find other tips as well.
+If you just followed the steps in :doc:`dev-environment` for the first time, you will need to get set up to deploy code to Payara. Below you'll find other tips as well.
```
Comment (scolapasta) [inline doc/sphinx-guides/source/developers/tips.rst:5]: your app server

File: doc/sphinx-guides/source/developers/troubleshooting.rst [hunk 11]
```diff
@@ -67,7 +67,7 @@ mail.smtp.socketFactory.class			javax.net.ssl.SSLSocketFactory
 
 **\*WARNING**: Entering a password here will *not* conceal it on-screen. It’s recommended to use an *app password* (for smtp.gmail.com users) or utilize a dedicated/non-personal user account with SMTP server auths so that you do not risk compromising your password.
 
-Save these changes at the top of the page and restart your Glassfish server to try it out.
+Save these changes at the top of the page and restart your Payara server to try it out.
```
Comment (scolapasta) [inline doc/sphinx-guides/source/developers/troubleshooting.rst:70]: app server

File: doc/sphinx-guides/source/developers/troubleshooting.rst [hunk 12]
```diff
@@ -76,7 +76,7 @@ The mail session can also be set from command line. To use this method, you will
 
 These properties can be tailored to your own preferred mail service, but if all else fails these settings work fine with Dataverse development environments for your localhost.
 
-+ If you're seeing a "Relay access denied" error in your Glassfish logs when your app attempts to send an email, double check your user/password credentials for the Mail Host you're using.
++ If you're seeing a "Relay access denied" error in your Payara logs when your app attempts to send an email, double check your user/password credentials for the Mail Host you're using.
```
Comment (scolapasta) [inline doc/sphinx-guides/source/developers/troubleshooting.rst:79]: app server

File: doc/sphinx-guides/source/installation/advanced.rst [hunk 13]
```diff
@@ -7,22 +7,22 @@ Advanced installations are not officially supported but here we are at least doc
 .. contents:: |toctitle|
 	:local:
 
-Multiple Glassfish Servers
---------------------------
+Multiple Payara Servers
+-----------------------
```
Comment (scolapasta) [inline doc/sphinx-guides/source/installation/advanced.rst:11]: app server

Comment (scolapasta) [reply doc/sphinx-guides/source/installation/advanced.rst:11]: and all the references in this section below could be changed as well (except obviously the directory names

File: doc/sphinx-guides/source/installation/oidc.rst [hunk 14]
```diff
@@ -78,7 +78,7 @@ Now load the configuration into Dataverse using the same API as with :doc:`oauth
 
 ``curl -X POST -H 'Content-type: application/json' --upload-file my-oidc-provider.json http://localhost:8080/api/admin/authenticationProviders``
 
-Dataverse will automatically try to load the provider and retrieve the metadata. Watch the Glassfish log for errors.
+Dataverse will automatically try to load the provider and retrieve the metadata. Watch the Payara log for errors.
```
Comment (scolapasta) [inline doc/sphinx-guides/source/installation/oidc.rst:81]: app server

File: doc/sphinx-guides/source/installation/r-rapache-tworavens.rst [hunk 15]
```diff
@@ -291,7 +293,7 @@ Dataverse URL         ``http://{your hostname}:8080``     URL of the Dataverse t
 Please note the default values above. The installer assumes 
 
 - that you are running both the Dataverse and TwoRavens/rApache on the same host; 
-- the default ports for Apache (80) and Glassfish that is serving your Dataverse (8080); 
+- the default ports for Apache (80) and Payara that is serving your Dataverse (8080);
```
Comment (scolapasta) [inline doc/sphinx-guides/source/installation/r-rapache-tworavens.rst:296]: app server

File: doc/sphinx-guides/source/installation/r-rapache-tworavens.rst [hunk 16]
```diff
@@ -330,7 +332,7 @@ Compare the two files. **It is important that the two copies are identical**.
 
 **If different**: 
 
-- the **TwoRavens version wins**. Meaning, you need to copy the version supplied with this TwoRavens distribution and overwrite the Glassfish version (above); then restart Glassfish. 
+- the **TwoRavens version wins**. Meaning, you need to copy the version supplied with this TwoRavens distribution and overwrite the Payara version (above); then restart Payara.
```
Comment (scolapasta) [inline doc/sphinx-guides/source/installation/r-rapache-tworavens.rst:335]: app server

File: doc/sphinx-guides/source/installation/r-rapache-tworavens.rst [hunk 17]
```diff
@@ -368,7 +370,7 @@ If the file does NOT appear as Tabular Data - if it is shown as Stata/dta,
 and no tabular attributes - the numbers of Variables and Observations and the UNF - 
 are being displayed, try to refresh the page a couple of times. If that doesn't 
 change the view to Tabular, it likely means that something went very wrong with the 
-tabular ingest. Consult the Glassfish server log for any error messages that may 
+tabular ingest. Consult the Payara server log for any error messages that may
```
Comment (scolapasta) [inline doc/sphinx-guides/source/installation/r-rapache-tworavens.rst:373]: app server

File: doc/sphinx-guides/source/installation/r-rapache-tworavens.rst [hunk 18]
```diff
@@ -418,9 +420,9 @@ You should get the output that looks like this::
         ...
 
 If you are getting an error message instead, this is likely an Rserve connection problem. 
-Consult the Glassfish server log for any Rserve-related "connection refused" messages. 
+Consult the Payara server log for any Rserve-related "connection refused" messages.
```
Comment (scolapasta) [inline doc/sphinx-guides/source/installation/r-rapache-tworavens.rst:423]: app server

File: doc/sphinx-guides/source/installation/r-rapache-tworavens.rst [hunk 19]
```diff
@@ -418,9 +420,9 @@ You should get the output that looks like this::
         ...
 
 If you are getting an error message instead, this is likely an Rserve connection problem. 
-Consult the Glassfish server log for any Rserve-related "connection refused" messages. 
+Consult the Payara server log for any Rserve-related "connection refused" messages. 
 See if Rserve is running, and start it with ``service rserve start``, if necessary. 
-Check if the Rserve host name, username and password in the Glassfish configuration match 
+Check if the Rserve host name, username and password in the Payara configuration match
```
Comment (scolapasta) [inline doc/sphinx-guides/source/installation/r-rapache-tworavens.rst:425]: app server

**Comment by scolapasta** [comment]
One more addition: we should mention the jsf patch for those who will manually update their environment. (unless I just missed it)

File: doc/sphinx-guides/source/admin/harvestclients.rst (hunk 1 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/admin/harvestclients.rst]: Fixed in b008f454b.

File: doc/sphinx-guides/source/admin/metadataexport.rst (hunk 2 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/admin/metadataexport.rst:31]: Fixed in db70a247c.

File: doc/sphinx-guides/source/admin/monitoring.rst (hunk 3 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/admin/monitoring.rst:95]: Fixed in 4dca56ff3.

File: doc/sphinx-guides/source/admin/monitoring.rst (hunk 4 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/admin/monitoring.rst:98]: Fixed in 4dca56ff3.

File: doc/sphinx-guides/source/admin/troubleshooting.rst (hunk 5 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/admin/troubleshooting.rst:107]: Fixed in 0e59c492b (fixed "symptomps" too).

File: doc/sphinx-guides/source/admin/troubleshooting.rst (hunk 6 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/admin/troubleshooting.rst:35]: Cool, me too.

File: doc/sphinx-guides/source/developers/coding-style.rst (hunk 7 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/developers/coding-style.rst:88]: Fixed in 7bdb809d1.

File: doc/sphinx-guides/source/developers/debugging.rst (hunk 8 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/developers/debugging.rst:11]: Fixed in cffaf5a8f.

File: doc/sphinx-guides/source/developers/testing.rst (hunk 9 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/developers/testing.rst:171]: Fixed in 7119c2c2a.

File: doc/sphinx-guides/source/developers/tips.rst (hunk 10 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/developers/tips.rst:5]: Fixed in 95ebcb41b.

File: doc/sphinx-guides/source/developers/troubleshooting.rst (hunk 11 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/developers/troubleshooting.rst:70]: Fixed in 0e1e364e7.

File: doc/sphinx-guides/source/developers/troubleshooting.rst (hunk 12 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/developers/troubleshooting.rst:79]: Fixed in e817dcf28.

File: doc/sphinx-guides/source/installation/advanced.rst (hunk 13 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/installation/advanced.rst:11]: Fixed in c717d63a0.

File: doc/sphinx-guides/source/installation/oidc.rst (hunk 14 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/installation/oidc.rst:81]: Fixed in fa205f0bc.

File: doc/sphinx-guides/source/installation/r-rapache-tworavens.rst [hunk 20]
```diff
@@ -78,7 +80,7 @@ server to download the file **directly from the Dataverse application**. Access
 URLs need to be configured for this to work properly (this is done by the TwoRavens 
 installer script in step ``3.``)  
 
-If you install all components on a single server and front Glassfish with Apache 
+If you install all components on a single server and front Payara with Apache
```
[the remark this answers is not in the document]
Comment (pdurbin) [reply doc/sphinx-guides/source/installation/r-rapache-tworavens.rst:83]: @scolapasta before changing this one and the rest below in `r-rapache-tworavens.rst` I'd like to check with @djbrooke and @landreev about the plan for maintaining this "How to install TwoRavens" documentation. We don't even use Two Ravens ourselves anymore.

As I wrote under "Special notes for your reviewer",

'In the Installation Guide, should we delete the old Two Ravens content? I put a note saying "Please note that in the text below, Glassfish was changed to Payara but not tested."'

File: doc/sphinx-guides/source/installation/r-rapache-tworavens.rst (hunk 15 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/installation/r-rapache-tworavens.rst:296]: See note above about `r-rapache-tworavens.rst`.

File: doc/sphinx-guides/source/installation/r-rapache-tworavens.rst (hunk 16 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/installation/r-rapache-tworavens.rst:335]: See note above about `r-rapache-tworavens.rst`.

File: doc/sphinx-guides/source/installation/r-rapache-tworavens.rst (hunk 17 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/installation/r-rapache-tworavens.rst:373]: See note above about `r-rapache-tworavens.rst`.

File: doc/sphinx-guides/source/installation/r-rapache-tworavens.rst (hunk 18 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/installation/r-rapache-tworavens.rst:423]: See note above about `r-rapache-tworavens.rst`.

File: doc/sphinx-guides/source/installation/r-rapache-tworavens.rst (hunk 19 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/installation/r-rapache-tworavens.rst:425]: See note above about `r-rapache-tworavens.rst`.

File: doc/sphinx-guides/source/installation/r-rapache-tworavens.rst [hunk 21]
```diff
@@ -444,7 +446,7 @@ Symptom: the variables view is initialized properly, but no model output appears
 I. Ports configuration discussion
 ---------------------------------
 
-By default, Glassfish will install itself on ports 8080 and 8181 (for
+By default, Payara will install itself on ports 8080 and 8181 (for
```
[the remark this answers is not in the document]
Comment (pdurbin) [reply doc/sphinx-guides/source/installation/r-rapache-tworavens.rst:449]: See note above about `r-rapache-tworavens.rst`.

File: doc/sphinx-guides/source/installation/r-rapache-tworavens.rst [hunk 22]
```diff
@@ -458,16 +460,16 @@ defaults, and you should have a working installation in no
 time. However, if you are planning to use this installation to
 actually serve data to real users, you will most likely want to run your Dataverse 
 on a standard port; and to use ``HTTPS``. It is definitely possible to configure 
-Glassfish to serve the application under ``HTTPS`` on port 443. However, we 
-**do not recommend** this setup! For at least 2 reasons: 1. Running Glassfish on 
+Payara to serve the application under ``HTTPS`` on port 443. However, we
```
[the remark this answers is not in the document]
Comment (pdurbin) [reply doc/sphinx-guides/source/installation/r-rapache-tworavens.rst:463]: See note above about `r-rapache-tworavens.rst`.

File: doc/sphinx-guides/source/installation/shibboleth.rst [hunk 23]
```diff
@@ -28,7 +28,7 @@ Only Red Hat Enterprise Linux (RHEL) and derivatives such as CentOS have been te
 Install Apache
 ~~~~~~~~~~~~~~
 
-We will be "fronting" Glassfish with Apache so that we can make use of the ``mod_shib`` Apache module. We will also make use of the ``mod_proxy_ajp`` module built in to Apache.
+We will be "fronting" Payara with Apache so that we can make use of the ``mod_shib`` Apache module. We will also make use of the ``mod_proxy_ajp`` module built in to Apache.
```
[the remark this answers is not in the document]
Comment (pdurbin) [reply doc/sphinx-guides/source/installation/shibboleth.rst]: Fixed in 343437d0e.

File: doc/sphinx-guides/source/installation/shibboleth.rst [hunk 24]
```diff
@@ -65,26 +65,17 @@ Please note that during the installation it's ok to import GPG keys from the Shi
...
-
-For more background on the patch, please see https://java.net/jira/browse/GRIZZLY-1787 and https://github.com/IQSS/dataverse/issues/2180 and https://github.com/payara/Payara/issues/350
-
-This problem has been reported to Glassfish at https://java.net/projects/glassfish/lists/users/archive/2015-07/message/1 and while Glassfish 4.1.1 includes a new enough version of Grizzly to fix the bug, other complicating factors prevent its adoption (look for "Glassfish 4.1.1" in the :doc:`prerequisites` section for details on why it is not recommended).
+Configure Payara
+----------------
 
-Glassfish HTTP and HTTPS ports
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Payara HTTP and HTTPS ports
```
[the remark this answers is not in the document]
Comment (pdurbin) [reply doc/sphinx-guides/source/installation/shibboleth.rst]: Fixed in 343437d0e.

File: doc/sphinx-guides/source/installation/shibboleth.rst [hunk 25]
```diff
@@ -305,7 +298,7 @@ The JSON in ``DiscoFeed`` comes from the list of IdPs you configured in the ``Me
 Add the Shibboleth Authentication Provider to Dataverse
 -------------------------------------------------------
 
-Now that you've configured Glassfish, Apache, and ``shibd``, you are ready to turn your attention back to Dataverse to enable Shibboleth as an "authentication provider." You will be using ``curl`` to POST the `following JSON file <../_static/installation/files/etc/shibboleth/shibAuthProvider.json>`_ to the ``authenticationProviders`` endpoint of the :doc:`/api/native-api`.
+Now that you've configured Payara, Apache, and ``shibd``, you are ready to turn your attention back to Dataverse to enable Shibboleth as an "authentication provider." You will be using ``curl`` to POST the `following JSON file <../_static/installation/files/etc/shibboleth/shibAuthProvider.json>`_ to the ``authenticationProviders`` endpoint of the :doc:`/api/native-api`.
```
[the remark this answers is not in the document]
Comment (pdurbin) [reply doc/sphinx-guides/source/installation/shibboleth.rst]: Fixed in 343437d0e.

File: doc/sphinx-guides/source/installation/r-rapache-tworavens.rst (hunk 20 above)
[the remark this answers is not in the document]
Comment (djbrooke) [reply doc/sphinx-guides/source/installation/r-rapache-tworavens.rst:83]: Thanks for tagging me and for bringing this up. Let me check into whether or not we can officially drop support. I'll have an answer later this week. 

I don't necessarily want to hold up this PR, so let's just make the changes to Payara. If we decide to delete this section I'll create a new issue and we can follow with a later PR.

**Comment by pdurbin** [comment]
> we should mention the jsf patch for those who will manually update their environment

I just added a line about the JSF patch in b3619996c.

**Comment by pdurbin** [comment]
I added a ton more "app server" references, as requested by @scolapasta 

These are the open questions I have for reviewers:

- In the Installation Guide, should we delete the old Two Ravens content? I put a note saying "Please note that in the text below, Glassfish was changed to Payara but not tested."
- When should we update Vagrant code?
- Should we clear out the "Containers" page in the dev guide and delete the related scripts, Openshift config and Docker files? The content is old and references Glassfish.

File: doc/sphinx-guides/source/installation/r-rapache-tworavens.rst (hunk 20 above)
[the remark this answers is not in the document]
Comment (pdurbin) [reply doc/sphinx-guides/source/installation/r-rapache-tworavens.rst:83]: @djbrooke sorry, I'm confused. The changes to Payara have already been made. Do you want me to change those lines to "app server"?

[the remark this answers is not in the document]
Comment (djbrooke) [reply doc/sphinx-guides/source/installation/r-rapache-tworavens.rst:83]: @pdurbin yes, I was confused (Github branch navigation issues :)), let's just use "app server" and be done with this for now. Thank you!

[the remark this answers is not in the document]
Comment (pdurbin) [reply doc/sphinx-guides/source/installation/r-rapache-tworavens.rst]: @djbrooke ok, done in 6607223f9. Also, while you're thinking about TwoRavens docs, here are a couple related issues: #4429 and #4136.

**Comment by pdurbin** [comment]
We discussed this pull request in tech hours and decided a few things.

- I'd like to update (or possibly delete) the Vagrant stuff, which has references to Glassfish 4.1.
- I'd like to delete the aging OpenShift content and files, which references Glassfish 4.1.
- We'll let @pameyer update the docker-aio/docker-dcm content when he's ready (no rush!).
- I'd like @landreev to take a look at the "run the installer" content in the Installation Guide. I made a few tweaks to it (making the new Python installer the default). @donsizemore mentioned that it should be written with an eye toward CentOS or RHEL rather than a laptop.

The other day in Slack we discussed Two Ravens.

- @djbrooke will give us guidance if we can delete the "how to install TwoRavens" content, which had a lot of references to Glassfish.

If there are other things I'm forgetting, please let me know.

**Pull request merged**